app/components/trainees/confirmation/degrees/view.html.erb

```diff
@@ -1,11 +1,14 @@
 <% if degrees.present? %>
   <% degrees.each do |degree| %>
-    <%= render_component SummaryCard::View.new(
-        title: degree_title(degree),
-        heading_level: 2,
-        rows: get_degree_rows(degree)
-      )
-    %>
+    <%= render_component SummaryCard::View.new(title: degree_title(degree), heading_level: 2, rows: get_degree_rows(degree)) do |component| %>
+      <% if show_delete_button %>
+        <% component.with(:header_actions) do %>
+          <%= form_with model: [trainee, degree], method: :delete, local: true, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
```

Comment:
Is there a way to set this `GOVUKDesignSystemFormBuilder::FormBuilder` like 

https://github.com/DFE-Digital/register-trainee-teachers/blob/b0acf6df79d2954e26851ec6336bd9d9df0a1f67/app/controllers/application_controller.rb#L9

Reply:
i've just used the native approach similar to one of the other components https://github.com/DFE-Digital/register-trainee-teachers/pull/229/commits/fb4e55ecc56e03c36a11f80f0eb8b21d80888075